pkg/prow/job_test.go

```diff
@@ -17,16 +17,28 @@
 package prow
 
 import (
+	"errors"
```

Comment:
format the imports list based on https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#imports

Reply:
fixed~